stylesheets/base/_typography.scss

```diff
@@ -32,21 +35,33 @@ i {
  * Links
  */
 
+/** Links are underlined by default
+ * On link focus, hover, and active events:
+ * 1. Change the color
+ * 2. Show mouse cursor as pointer on hover
+ * 3. Remove link underline
+ */
 a {
   color: $flame-orange;
-  text-decoration: underline; // underline by default
+  text-decoration: underline;
 
   @include on-event {
-    color: $crusta-orange;
-    cursor: pointer;
-    text-decoration: none;
+    color: $crusta-orange; /* 1 */
+    cursor: pointer; /* 2 */
+    text-decoration: none; /* 3 */
   }
 }
 
+/**
+ * Make a link bold
+ */
 .link--bold {
   font-weight: $font-weight-bold;
 }
 
+/**
+ * Underline a link
```

Comment:
Shouldn't this be "Remove link underline"?

Also wondering if this style shouldn't show the link on hover?

Reply:
Hmmm that's a good question. I was thinking primarily about the latter use case (nav links) because buttons have separate classes. But I think classes which control colors should probably be wrapped in the component, right? Unless it makes sense to do an `unset` here?

Reply:
Yeah okay I'm with that. Let me push up a commit because I also need to pull that out of the stories.